packages/wallets/src/translations/i18n.ts

```diff
@@ -44,6 +53,13 @@ i18n.use(initReactI18next).init({
     fallbackLng: 'EN',
     lng: currentLanguage,
     ns: ['translations'],
+    react: {
+        hashTransKey(defaultValue: string) {
+            // FIX: replace this temporary key finding until we have a better key generation
+            return Object.entries(EN).find(([, value]) => value === defaultValue)?.[0] ?? defaultValue;
```

Comment:
This is to be fixed in this PR: https://github.com/binary-com/deriv-app/pull/11898